chain/client/src/info.rs

```diff
@@ -256,6 +258,15 @@ impl InfoHelper {
     }
 }
 
+fn extra_telemetry(client_config: &ClientConfig) -> serde_json::Value {
```

Comment:
nit: do we need some tests to ensure that the `blob` is properly initialized?

Reply:
Added